doc/pact_testing.md

```diff
@@ -42,19 +42,15 @@ file to the broker as `branch-<branch-name>` using the `pact:publish:branch`
 rake task. It then checks out the master branch of publishing-api and verifies
 the pact against that branch using the `pact:verify:branch` rake task.
 
-When a new version of gds-api-adapters is released by bumping the version, the
-job will also publish the pact as the released version, by running the
-`pact:publish:released_version` rake task.
-
 In publishing-api itself, the Jenkins job runs the `pact:verify` rake task to
-verify the current branch against both master and that released version pact.
+verify the current branch against the master branch.
 
 
 ## Running the pacts in development
 
 You can use `pact:verify` locally to run the current branch against the master
-and released version pacts stored in Pact Broker. If you need to run them
-against a local version of gds-api-adapters, run the tests in that directory
+branch of pacts stored in Pact Broker. If you need to run them against a local
+version of gds-api-adapters, run the tests in that directory
 and then set the `USE_LOCAL_PACT` env variable:
 
     USE_LOCAL_PACT=1 bundle exec rake pact:verify
```

Comment:
Looks like we've removed this altogether too - maybe just remove this section from the docs? I can't see where we use `USE_LOCAL_PACT` in the code.

Reply:
It's used here: https://github.com/alphagov/publishing-api/blob/4e3ef4f4e3bef61c5ecea0199e0ede5a8300d4b1/spec/service_consumers/pact_helper.rb#L21-L23